src/interpolaters/Interpolate_transform.re

```diff
@@ -0,0 +1,42 @@
+let interpolateTransform = (~range as (rl, rh), ~domain as (dl, dh), ~value) => {
+  let fromTransform = Parse_transform.parseTransform(dl);
+  let toTransform = Parse_transform.parseTransform(dh);
+
+  let transforms =
+    switch (
+      fromTransform.transform->Js.Nullable.toOption,
+      toTransform.transform->Js.Nullable.toOption,
+    ) {
+    | (Some(tl), Some(th)) =>
+      let tl_1 = Js.String.split(", ", tl);
```

Comment:
Careful here 😅 the white space is optional can be repeated and you could have new lines. Time for another regex?

Reply:
Right, that's a good shout, so we need to support `translate(10px,10px)` as well. Can you give a newline example? Do you mean we could get something like:

```js
from: {
   transform: `translate(20px,
     20px)`
}

I could see just splitting by the comma and then trimming the whitespace in lieu of another regex here, but curious to hear your thoughts.

Reply:
The no whitespace case should be addressed by this commit: https://github.com/FormidableLabs/renature/pull/42/commits/a88260ca4c00a783d40a834d613c1bdb3bb488f6